tinynn/converter/operators/torch/aten.py

```diff
@@ -2388,3 +2388,18 @@ def parse(self, node, attrs, args, graph_converter):
 
         self.run(node)
         ATenToOperator.parse_common(self, node, attrs, args, graph_converter)
+
+
+class ATenTopkOperator(ATenTopkSchema):
+    def parse(self, node, attrs, args, graph_converter):
+        super().parse(node, attrs, args, graph_converter)
+
+        self.run(node)
+        input_tensor, k, dim, largest, sorted = self.input_tensors[:5]
+        assert dim == -1 and largest is True and sorted is True, "tflite topk only dim=-1,largest=True and sorted=True"
```

Comment:
Please use `in (1, True)` or remove `== True` for boolean comparisons. Also, you should also accept positive indices.
```suggestion
        assert dim in (-1, input_tensor.ndim - 1) and largest in (1, True) and sorted in (1, True), "tflite topk only dim=-1, largest=True and sorted=True"
```